packages/common/src/sites/_internal/SiteSchema.ts

```diff
@@ -0,0 +1,64 @@
+import { IConfigurationSchema } from "../../core";
+import {
+  ENTITY_ACCESS_SCHEMA,
+  ENTITY_CATEGORIES_SCHEMA,
+  ENTITY_NAME_SCHEMA,
+  ENTITY_TAGS_SCHEMA,
+} from "../../core/schemas/shared";
+
+export type SiteEditorType = (typeof SiteEditorTypes)[number];
+export const SiteEditorTypes = ["hub:site:edit"] as const;
+
+/**
+ * defines the JSON schema for a Hub Site's editable fields
+ */
+export const SiteSchema: IConfigurationSchema = {
+  required: ["name"],
+  type: "object",
+  properties: {
+    name: ENTITY_NAME_SCHEMA,
+    summary: {
+      type: "string",
+    },
+    description: {
+      type: "string",
+    },
+    access: ENTITY_ACCESS_SCHEMA,
+    groups: {
+      type: "array",
+      items: {
+        type: "string",
+      },
+    },
+    // status: {
+    //   type: "string",
+    //   default: PROJECT_STATUSES.notStarted,
+    //   enum: Object.keys(PROJECT_STATUSES),
+    // },
+    location: {
+      type: "object",
+    },
+    tags: ENTITY_TAGS_SCHEMA,
+    categories: ENTITY_CATEGORIES_SCHEMA,
+    view: {
+      type: "object",
+      properties: {
+        featuredContentIds: {
```

Comment:
might also want to abstract this as a shared schema if it's going to be reused between entities?

Reply:
I have no idea - I just copy-pasted stuff from Project... just to get things working quickly. The SiteSchema is definitely pre-alpha level of completeness :) I'll remove for now